scripts/lib/CIME/XML/batch.py

```diff
@@ -111,13 +111,13 @@ def get_batch_jobs(self):
         and the second a dict of qualifiers for the job
         """
         jobs = []
-        bnode = self.get_node("batch_jobs")
-        for jnode in bnode:
-            if jnode.tag == "job":
-                name = jnode.get("name")
+        bnode = self.get_child("batch_jobs")
+        for jnode in self.get_children(root=bnode, no_validate=True):
+            if self.name(jnode) == "job":
+                name = self.get(jnode, "name")
                 jdict = {}
-                for child in jnode:
-                    jdict[child.tag] = child.text
+                for child in self.get_children(root=jnode, no_validate=True):
```

Comment:
Don't need to validate this get_children call since this is a new get_children call and not a replacement of an existing call.